### DISCUSSION TITLE
Did anyone else get spam on a GitHub-specific email address?

### DISCUSSION THREAD

**Discussion by lgommans** [community/community#22834] [New to GitHub]
<p>I just (8 minutes ago at the moment of posting excluding edits) received spam on an email address that I only ever used for GitHub, and I’m fairly sure that I never attached it to my git commits (for committing I specifically use <code>git-client@lgms.nl</code> and blocked it from receiving emails, and website-made GitHub commits show up as <code>lgommans@users.noreply.github.com</code>).</p>
<p>The “To” field in the mail headers is set to a completely different address (someone at a primary school in the Netherlands), but in the mail server logs I can see that the envelope is (rot13’d then base64’d) dHZndWhvLnBiekB5aHB0Ynp6bmFmLmF5, i.e. the email address on my GitHub account. I’ve changed it to something else (involving a csprng) so it can’t be coincidence in the future. At least one other active GitHub account of mine is so far unaffected (I haven’t checked work accounts yet).</p>
<p>This was before registering for <code>github.community</code> which requires access to my email addresses, so the issue wasn’t here.</p>
<p>The spam was titled <code>Super intense welcome offer up to €888</code> and the content was just a big picture, showing some coins, games, a lady, and the whole thing is a clickable link. Of course, this can be different for everyone, but if it’s a similar theme with similar timing and a similar source, then it might be more likely to be the same originating list.</p>
<p>For now, I guess we’ll assume coincidence, and it’ll be hard to find other people who run their own mail server and track this down… but then GitHub is large and for developers, so hopefully anyone else who finds this also finds this thread and can confirm that GitHub / Microsoft has had a data breach.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome to the community!</p>
<p>We’re certainly not aware of any breaches. If, however, you have a Google search for that email address, I found one of yours posted publicly in an issue for a popular project hosted on GitHub.  I don’t want to post that here, as it would further expose that email.</p>
<p>If you can’t find it, I’ll happily email you the link. I hope that puts your mind at rest a bit!</p>

**Reply by BlackFrog1** [reply]
I am getting a lot of spam this morning.  


![Screenshot 2023-11-07 at 4 32 26 AM](https://github.com/community/community/assets/550820/e225779d-11c3-49da-87c8-96ce9ced6171)


Here is the header from the message 
![Screenshot 2023-11-07 at 4 35 31 AM](https://github.com/community/community/assets/550820/d96e2c5e-2084-4499-a64c-e55508c8ad5b)

**Comment by lgommans** [comment]
<p>Thanks canuckjacq for looking into this! That clears it up then :slight_smile: Sorry for the bother!</p>
